Projects/App/Sources/UI/Place/PlaceList/View/PlaceListViewController.swift

```diff
@@ -42,10 +120,51 @@ extension PlaceListViewController {
         tableView.separatorStyle = .none
     }
     
+    private func removeBackgroundAndDivider() {
+        let removeBackgroundDivider = UIImage()
+        segmentedControl.setBackgroundImage(removeBackgroundDivider, for: .normal, barMetrics: .default)
+        segmentedControl.setBackgroundImage(removeBackgroundDivider, for: .selected, barMetrics: .default)
+        segmentedControl.setDividerImage(removeBackgroundDivider, forLeftSegmentState: .selected, rightSegmentState: .normal, barMetrics: .default)
+    }
+    
     private func createLayout() {
-        view.addSubview(tableView)
+        view.addSubviews([headerView, segmentedControl, segmentIndicator, questionMarkImage, tableView])
+        headerView.addSubviews([segmentedControl, segmentIndicator, questionMarkImage, sideButtonImage])
+        
+        headerView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
+            make.horizontalEdges.equalToSuperview()
+            make.height.equalTo(85)
+        }
+        
+        segmentedControl.snp.makeConstraints { make in
+            make.verticalEdges.equalToSuperview().inset(27)
+            make.left.equalToSuperview().inset(20)
+            make.width.equalTo(172)
+        }
+        
+        segmentIndicator.snp.makeConstraints { make in
+            make.top.equalTo(segmentedControl.snp.bottom).offset(3)
+            make.height.equalTo(3)
+            make.width.equalTo(20)
+            make.centerX.equalTo(segmentedControl.snp.centerX).dividedBy(segmentedControl.numberOfSegments)
+        }
+        
+        questionMarkImage.snp.makeConstraints { make in
+            make.left.equalTo(segmentedControl.snp.right)
+            make.top.equalTo(segmentedControl.snp.top)
+            make.width.equalTo(21)
+            make.height.equalTo(21)
+        }
+        
+        sideButtonImage.snp.makeConstraints { make in
+            make.verticalEdges.equalToSuperview().inset(10)
+            make.right.equalToSuperview()
+        }
+        
         tableView.snp.makeConstraints { make in
-            make.edges.equalToSuperview()
+            make.top.equalTo(headerView.snp.bottom).offset(20)
```

Comment:
p1;

여기 offset 때문에 간격이 너무 넓네요 없애주세요!

```suggestion
            make.top.equalTo(headerView.snp.bottom)
```

Reply:
7281c930c08eb29f3a068d666e5166dd3e56c9bc 변경하였습니다!